src/constants/ConstThemes.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable import/prefer-default-export */
-export const THEMES = {
+export const ConstantThemes = {
```

Comment:
Please revert this. The files are placed in constant folder that depicts that these are constants.

Reply:
Sure @Abhishek-kumar09  :)

Reply:
@Abhishek-kumar09 I have made changes and I didn't change the `constTheme.js` to `theme.js` in constants dir because we already have a file name called `theme.js  ` :)